<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManufacturerDialog</class>
 <widget class="QDialog" name="ManufacturerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление производителя</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(199, 199, 199);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QGroupBox" name="Group3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 220);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtCountry"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Страна происхождения</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Название</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="btnOk">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(128, 128, 128);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="btnCancel">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(128, 128, 128);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
